src/assets/asset-transfers.md

```diff
@@ -118,7 +121,7 @@ curl -X GET "https://service.centrapay.com/api/asset-transfers/M7Kn2stAxNa6ri7h"
   "senderAccountId": "aBc932S9182qwCDqwer",
   "recipientAccountId": "oS3Xom2au3Ooy9aihai",
   "createdAt": "2020-05-01T12:30:00.000Z",
-  "sentAt": "2020-05-01T12:30:00.000Z"
+  "updatedAt": "2020-05-02T01:03:37.222Z"
```

Comment:
We never implemented sentAt, but it can be inferred.

We can come back to add this if we need it later.